scripts/radar.js

```diff
@@ -0,0 +1,11 @@
+// Description
+//   Gets the Beureau of Meteorology's latest radar image
+// Commands
+//   !`radar` - Gets Brisbane's radar
+//
+
+module.exports = function (robot) {
+	robot.respond(/!?radar/i, function (res) {
+    res.send("https://bom.lambda.tools/?location=brisbane&_cache=" + Date.now());
```

Comment:
I think there should be a comment here linking to the docs for this in case it breaks and we don't know why